### PR TITLE
Minor filepath and folder command changes in powershell script

### PR DESCRIPTION
**Pull request by Nikolaj-K**
I went back to a Windows 10 installation attempt from May (https://github.com/Chia-Network/chia-blockchain/pull/5158). 
This PR is now some small changes that are improvements in my Windows install (both also among the issues mentioned back then on the side), the first enables running the powershell script repeatedly, ignoring a `mkdir` "already exists" complaint, and the second is using another command for jumping back two directories, where the one currently used (with the standalone dash symbol) is only a few years old. 

edit: I try to keep the branch synced with dev once a week.

edit: Removed further insights I had while trying to install on Windows, from this PR description, since it seems to result in a tl;dr situation and is only loosely related to the changes (you can still look at them by going back 2 edits of this description).

### CODE REVIEW

**Comment by hoffmang9** [comment]
We only have miniupnpc pre built for Python 3.8.

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: build_scripts/build_windows.ps1 [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 $ErrorActionPreference = "Stop"
 
-mkdir build_scripts\win_build
+mkdir build_scripts\win_build -ErrorAction SilentlyContinue
```
[the remark this answers is not in the document]
Comment (Nikolaj-K) [reply build_scripts/build_windows.ps1:5]: The link you posted currently seems to be dead. What I can say is that this sort of change was necessary to make that script run through without failure. There seemed to be no interest to make the code work on Windows, so I don't think I'll go though the install attempt again in the next few months. Feel free to directly take over any change that's helpful to other branches. Best.

[the remark this answers is not in the document]
Comment (cmmarslender) [reply build_scripts/build_windows.ps1:5]: the build_* scripts here are more for making the installers, which typically run in CI. If you are trying to just install on windows from source, I believe you'd want to just use this install script: https://github.com/Chia-Network/chia-blockchain/blob/main/Install.ps1

**Comment by emlowe** [comment]
Closing PR

**Pull request closed without merging**